lib/login.js

```diff
@@ -413,25 +413,24 @@ class Login {
 
   async _keepalive() {
     let req = {
-      method : 'GET',
-      url    : 'user/credits'
+      method: 'GET',
+      url: 'user/credits'
     };
     let response = await APIRequest(req);
     if (_.get(response, 'credits')) return response.credits;
     return false;
   }
 
   _pileSize() {
-    let data = this.__usermassinfo.pileSizeClientData.entries;
     return {
-      tradepile : data[0].value,
-      watchlist : data[2].value
+      tradepile: 100,
+      watchlist: 100
```

Comment:
This should not have fixed values.

Reply:
In FIFA 20 this values are fixed. There is no option to expand those.